src/taskFunction.js

```diff
@@ -0,0 +1,34 @@
+function createTaskElement(taskName, tasksLocal) {
+  // If tasksLocal is not already an array, initialize it as an empty array
+  if (!Array.isArray(tasksLocal)) {
+    tasksLocal = [];
+  }
+
+  const index = tasksLocal.length + 1;
+  const complete = false;
+  const taskString = { index, name: taskName, completed: complete };
+  tasksLocal.push(taskString);
+  localStorage.setItem('tasks', JSON.stringify(tasksLocal));
+}
+
+function arrangeIndexes(tasksLocal) {
+  for (let i = 0; i < tasksLocal.length; i += 1) {
+    tasksLocal[i].index = i + 1;
+  }
+}
+
+function deleteTaskElement(tasksLocal, taskIndex) {
+  tasksLocal = tasksLocal.filter((t) => t.index !== taskIndex);
+  arrangeIndexes(tasksLocal); // Reassign correct indexes after deletion
+  localStorage.setItem('tasks', JSON.stringify(tasksLocal));
+  document.location.reload();
+}
```

Comment:
- [ ] Great work on making the add and remove function :+1: but it would be nice if your page don't reload after adding or removing in order to follow the project requirements :+1: Kindly make a render method which would handle the task display :blush:

If you see this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) given by microverse then you won't feel any reloading from the app :thinking: